### PR TITLE
Latest changes to privacy policy

### PR DESCRIPTION
**Pull request by amansinghb**
Latest changes to privacy policy based on the DPA work

### CODE REVIEW

File: content/privacy.md [hunk 1]
```diff
@@ -15,9 +15,11 @@ We collect certain information or data about you when you use Pension Wise.
 
 During a Pension Wise guidance appointment questions will be asked about your personal and family affairs, financial arrangements and plans and general state of health.
 
-These details will be shared with our Pension Wise partners to provide you with a summary of your appointment. They may also be shared anonymously with third parties to help the government analyse and improve the performance of the service.
+These details will be used to provide you with a summary of your appointment. They may used anonymously to help the goverment and third parties analyse and improve the performance of the service.
```
Comment (benbarnett) [inline content/privacy.md]: This paragraph has a lot of typos

File: content/privacy.md [hunk 2]
```diff
@@ -15,9 +15,11 @@ We collect certain information or data about you when you use Pension Wise.
...
 
-If shared in this way, your information would be used only for this purpose and will not be disclosed any further.
+We work with a market research company and may share your contact details for this purpose
```
Comment (benbarnett) [inline content/privacy.md]: Do we need a full stop here?

File: content/privacy.md [hunk 3]
```diff
@@ -15,9 +15,11 @@ We collect certain information or data about you when you use Pension Wise.
...
+
+If you chose to receive a digital version of the appointment summary, your name and email address will be shared with the Goverment Digital Service. If shared in this way, your information would be used only for this purpose and will not be disclosed any further.
```
Comment (benbarnett) [inline content/privacy.md]: Typos again on Government

Comment (benlovell) [inline content/privacy.md]: Typo for Government.

File: content/privacy.md (hunk 1 above)
Comment (benlovell) [inline content/privacy.md]: They may *be* used

Comment (benlovell) [inline content/privacy.md]: Also another typo for government.

**Comment by amansinghb** [comment]
@benlovell @benbarnett fixed those typos. Should be all good now.

File: content/privacy.md [hunk 4]
```diff
@@ -15,9 +15,11 @@ We collect certain information or data about you when you use Pension Wise.
 
 During a Pension Wise guidance appointment questions will be asked about your personal and family affairs, financial arrangements and plans and general state of health.
 
-These details will be shared with our Pension Wise partners to provide you with a summary of your appointment. They may also be shared anonymously with third parties to help the government analyse and improve the performance of the service.
+These details will be used to provide you with a summary of your appointment. They may be used anonymously to help the government and third parties analyse and improve the performance of the service.
 
-If shared in this way, your information would be used only for this purpose and will not be disclosed any further.
+We work with a market research company and may share your contact details for this purpose.
+
+If you chose to receive a digital version of the appointment summary, your name and email address will be shared with the Government Digital Service. If shared in this way, your information would be used only for this purpose and will not be disclosed any further.
```
Comment (amansinghb) [inline content/privacy.md]: @alexkouz @CassBonner what do you guys think?

**Comment by amansinghb** [comment]
@benbarnett this should be all good to go now

**Comment by benbarnett** [comment]
Nice. 🏌

**Pull request merged**